src/gui/newwizard/setupwizardcontext.cpp

```diff
@@ -62,7 +62,16 @@ void SetupWizardContext::resetAccountBuilder()
 
 CoreJob *SetupWizardContext::startFetchUserInfoJob(QObject *parent) const
 {
-    return _accountBuilder.authenticationStrategy()->makeFetchUserInfoJobFactory(_accessManager).startJob(_accountBuilder.serverUrl(), parent);
+    const QUrl serverUrl = [this]() {
```

Comment:
Wouldn't a  x.serverUrl() function make sense? The url could be needed in other places too.